others/make_deb

```diff
@@ -127,15 +130,25 @@ if [[ ${branch_name} = "main" ]] || [[ ${branch_name} =~ ^[0-9]+\.[0-9]+\.[0-9]+
 # Debian file structure
 mkdir -p usr/local/lib
 if [ "${ADDITIONAL_LIB_NAME}" == "mqtt" ];  then
-    cp -R --preserve=links /usr/local/lib/libpaho* usr/local/lib
+    # For Raspberry Pi OS Bullseye
+    if [[ ${v_name} == *"bullseye"* ]]; then # Verify Ubuntu20.04 aarch64
+      if [[ ${architecture} == *"armv7l"* ]]; then
+        cp -R --preserve=links /usr/local/lib/arm-linux-gnueabihf/libpaho* usr/local/lib
+      else # aarch64
+        cp -R --preserve=links /usr/local/lib/aarch64-linux-gnu/libpaho* usr/local/lib
+      fi
+    else
+      cp -R --preserve=links /usr/local/lib/libpaho* usr/local/lib
+    fi
 fi
 if [ "${ADDITIONAL_LIB_NAME}" == "gcp" ];  then
     cp -R --preserve=links /usr/local/lib/libjwt* usr/local/lib
     cp -R --preserve=links /usr/local/lib/libjansson* usr/local/lib
 fi
 if [ "${ADDITIONAL_LIB_NAME}" == "iec" ];  then
-    # For Raspbian bullseye
-    if [[ ${os_name} == *"Raspbian"* && ${os_version} == *"11"* ]]; then
+    # For Raspberry Pi OS Bullseye
+    if [[ ${v_name} == *"bullseye"* ]]; then
+      # arch based check?
```

Comment:
As its packaging failed and a very old issue & we are not targeting iec plugins as such. So we can skip further on this.